lib/normalize.js

```diff
@@ -83,6 +83,27 @@ function normalizePackageBin (pkg, changes) {
   delete pkg.bin
 }
 
+function normalizePackageMan (pkg, changes) {
+  if (pkg.man) {
+    const mans = []
+    for (const man of (Array.isArray(pkg.man) ? pkg.man : [pkg.man])) {
+      if (typeof man !== 'string') {
+        changes?.push(`removed invalid "man [${man}]"`)
+      } else {
+        mans.push(secureRef(man))
+      }
+    }
+
+    if (!mans.length) {
+      changes?.push('empty "man" was removed')
+    } else {
+      pkg.man = mans
+      return pkg
```

Comment:
We don't actually use the return value here, returning `pkg` is probably not the right precedent.  I know we also do it in once case for `normalizePackageBin` and that one should probably be just a `return` also.

Reply:
Changing this should be a separate PR though. I don't want to block your wonderful work here on a nitpick. I'll approve and land this as-is.

Reply:
I can separate this part, np.

Reply:
It's up to you, you can fix it here too. Since the windows tests need fixing you'll be updating this PR anyways. I just didn't want to block a green PR over this.